frontend/packages/console-app/src/components/pdb/form-popover.scss

```diff
@@ -0,0 +1,16 @@
+.pdb-form-popover {
+    &__title {
+      margin-top: var(--pf-global--spacer--md);
+      margin-bottom: var(--pf-global--spacer--sm);
+    }
+    &__description {
+      font-size: var(--pf-global--FontSize--sm);
+    }
+  }
+
+  .pdb-form-availability-requirement {
+    margin-bottom: var(--pf-global--spacer--md);
+  }
```

Comment:
Don't think you need this one. The `Grid` (or if you switch to `Stack`) will handle this spacing.